examples/feature_selection/plot_f_test_vs_mi.py

```diff
@@ -39,11 +40,10 @@
 plt.figure(figsize=(15, 5))
 for i in range(3):
     plt.subplot(1, 3, i + 1)
-    plt.scatter(X[:, i], y)
+    plt.scatter(X[:, i], y, edgecolor='black', s=20)
```

Comment:
I somehow liked it more aesthetically with the black boundaries. should i reverse the change or can we keep it since this is already done?